app/services/vot/parser.rb

```diff
@@ -25,6 +26,36 @@ def self.no_sp_result
       def identity_proofing_or_ialmax?
         identity_proofing? || ialmax?
       end
+
+      def aal_level_requested
+        if aal2?
+          2
+        else
+          1
+        end
+      end
+
+      def ial_value_requested
+        if ialmax?
+          0
+        elsif identity_proofing?
+          2
+        else
+          1
+        end
+      end
+
+      def ial2_requested?
+        identity_proofing?
+      end
+
+      def ialmax_requested?
+        ialmax?
+      end
+
+      def piv_cac_requested?
+        hspd12?
```

Comment:
Your ticket is only about aal level requested - I would leave these to the tickets specific to those properties.

Reply:
Removed.